src/lib/idleTimeout.js

```diff
@@ -86,10 +90,13 @@ export const handleIdleTimeout = async (idleTimeoutConfig) => {
 		await store.setState({
 			idleTimeout: {
 				...store.state.idleTimeout,
-				idleWarningTimer: null,
-				idleTimeoutTimer: null,
 				idleTimeoutRunning: false,
 			},
+			idleTimeoutTimer: {
```

Comment:
@bhardwajaditya Can you please help me understand here why moving out `idleTimeoutTimer`? Can we directly persist `idleTimeout`? Did you find some issue with that?

Reply:
Actually persisting idle timeout was creating problems in case of multi tabs, we save the ID of set timeout and set interval as part of idleTimeout which was then used in clearTimers() function. In case of multi tabs one tab was overwritting changes of other tab and modifying this ID thus when clear tab was called only only tab timer was cleared.

I have added the IDs to non persistent data so that it remains tab specific and rest all remains same as per existing implementation

Reply:
Got it!! Thanks for the clarification @bhardwajaditya